src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -81,7 +81,6 @@ public abstract class AbstractTargetConfiguration implements TargetConfiguration
 
     private static final List<String> baseNativeImageArguments = Arrays.asList(
             "-Djdk.internal.lambda.eagerlyInitialize=false",
-            "-Ddebug.jdk.graal.jvmciConfigCheck=warn",
```

Comment:
shouldn't this be defined somewhere else then? (globally?)

Reply:
it is not needed once https://github.com/gluonhq/graal/pull/20 gets merged